### DISCUSSION TITLE
2SP-related stuff

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#246] [Suggestions]
### Describe the feature you'd like to see added

This game can certainly be more like the arcade, incl. credits display, title animation sequence using actual sprites, etc....
You can even revive the actual demo mode screens as well (lol, that might take a long time to code).

**Comment by emoose** [comment]
Spam above looks like a sitewide thing, weird https://www.google.com/search?q=%22In+the+installer+menu%2C+select%22+%22gcc.%22+site%3Agithub.com

**Comment by Sonic-TV** [comment]
idea: you can write some custom text on the area that supposed to display credits on 2SP mode and the title screen. (lol you can write a CREDIT(S) 0 there)

**Comment by Sonic-TV** [comment]
since the original issue's closed, I'll just put them here.
C2C needs to revive the animation on 5A(volcano) and 5D(rocket) endings when GOAL.
